build.sh

```diff
@@ -154,7 +154,7 @@ do
 done
 
 cmake -H. -B${dir} ${CMAKE_EXTRA_OPTIONS} -DCMAKE_BUILD_TYPE=RelWithDebInfo -DTESTS=ON -DCMAKE_INSTALL_PREFIX=..
-cmake --build ${dir} -- -j4
+cmake --build ${dir} -- -j20
```

Comment:
is this intentional?

Reply:
No, but it helps a lot if you have the cpus, I will revert that one.